static/src/javascripts/types/user-attributes.d.ts

```diff
@@ -0,0 +1,27 @@
+//the type of the response returned by https://members-data-api.theguardian.com/user-attributes/me/mma-monthlycontribution
```

Comment:
This link does not work

Reply:
yeah I'm not really checking these files, much of the content I've had nothing to do with, def more of a conversion process than anything else